src/input/datetime.rs

```diff
@@ -55,6 +54,7 @@ impl<'a> EitherDate<'a> {
 }
 
 #[cfg_attr(debug_assertions, derive(Debug))]
+#[derive(Clone)]
```

Comment:
i doubt this is required.

If you're changing, might be easier to replace with `#[derive(Debug, Clone)]` - I think `debug_assertions` was a folly.

Reply:
removed